backend-shared/hack/service_account.go

```diff
@@ -0,0 +1,170 @@
+package hack
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	rbacv1 "k8s.io/api/rbac/v1"
+	apierr "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/kubectl/pkg/scheme"
+	client "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	ArgoCDManagerServiceAccount     = "argocd-manager"
+	ArgoCDManagerClusterRole        = "argocd-manager-role"
+	ArgoCDManagerClusterRoleBinding = "argocd-manager-role-binding"
+)
+
+var (
+	hostname                          = "https://kubernetes.svc.default"
+	ArgoCDManagerNamespacePolicyRules = []rbacv1.PolicyRule{
```

Comment:
Using capital letters you actually export the variable, so other packages can `import hack` pkg. Is this intentional?

Reply:
Makes sense, whenever we export it in the future, we can always make it capital.